src/main/java/com/questhelper/helpers/quests/legendsquest/LegendsQuest.java

```diff
@@ -561,6 +579,9 @@ public void setupRequirements()
 
 		chargeOrbRunes = new ItemRequirements(LogicType.AND, "Runes for any charge orb spell you have the level to cast", cosmic3, elemental30);
 
+		teleToLegendsGuildHint = new ItemRequirement("Fairy ring BLR, or Ardougne Teleport then run north-east", -1, -1);
+		teleToJungleHint = new ItemRequirement("Fairy ring CKR then run south, or Brimhaven teleport then take Hajedy's cart to Shilo Village for 200 coins", -1, -1);
```

Comment:
Same as above hint. I'd say a default hint of maybe a glory if the player hasn't done Fairy Tale Part 2 or Shilo Village for the Karamja tele? Not sure if there's a better 'default' teleport for someone somehow doing this quest without having at least done Shilo.

Reply:
Shilo Village is a pre-requisite for Legends' Quest so I think using the cart is fine.
Adding an item suggestion is a bit trickier since it's not a one-step tele so I'll try solving it with a mix of words for the fairy-ring-less in [`2935827` (#1275)](https://github.com/Zoinkwiz/quest-helper/pull/1275/commits/293582762f226a17194228e39e28604ddb9ad40d)

So in the commit I recommend either a fairy ring staff if they have completed fairy ring p2, or 200gp + hint the player to use brimhaven tele or amulet of glory to get there. but no bank hint for the glory or brimhaven tele